docs/index.md

```diff
@@ -4,387 +4,50 @@ jupytext:
   text_representation:
     extension: .md
     format_name: myst
-kernelspec:
-  display_name: Python 3
-  language: python
-  name: python3
 
 # override the toc-determined page navigation order
-next-page: getting_started/flyte_fundamentals
-next-page-title: Flyte Fundamentals
+next-page: getting_started/quickstart_guide
```

Comment:
The "next" link is currently not rendering correctly (points to the wrong place), but I'm not sure why.

Reply:
We won't actually need this after monodocs is merged. Typically the page order is determined by the `toctree` directive. The `next-page`, `prev-page` thing is custom behavior that was meant to glue together the 5 separate docs sites.